src/main/java/me/moodcat/database/BulkInsertApp.java

```diff
@@ -0,0 +1,177 @@
+
+    /**
+     * Reference for injector in order to have concurrent transactions to our database.
+     */
+    private final AtomicReference<Injector> injectorAtomicReference = new AtomicReference<>();
+
+    /**
+     * The server.
+     */
+    private Server server;
+
+    public static void main(final String[] args) throws Exception {
+        new BulkInsertApp().run();
+    }
+
+    /**
+     * Run the bulk insertion.
+     *
+     * @throws Exception
+     *             when the bulkinsertion has failed.
+     */
+    public void run() throws Exception {
+        server = new Server(SERVER_PORT);
+        Runtime.getRuntime().addShutdownHook(new Thread(this::stopServer));
+        server.setHandler(attachHandlers());
+        server.start();
+        // Clear the database to avoid duplicates
+        injectorAtomicReference.get().getInstance(BulkInsertData.class).clear();
+
+        // Insert the data
+        injectorAtomicReference.get().getInstance(BulkInsertData.class).insertData();
+
+        // Insert rooms
+        injectorAtomicReference.get().getInstance(BulkInsertData.class)
+                .insertRandomRooms(NUMBER_OF_ROOMS);
+
+        // When done, stop the server
+        server.stop();
+    }
+
+    /**
+     * Stops the server.
+     */
+    public void stopServer() {
+        try {
+            this.server.stop();
+        } catch (final Exception e) {
+            log.error("Couldn't stop server.", e);
+        }
+    }
+
+    /**
+     * Create the handlers of the server.
+     * 
+     * @return the handlers.
+     */
+    public ContextHandlerCollection attachHandlers() {
+        final ContextHandlerCollection handlers = new ContextHandlerCollection();
+
+        MoodcatHandler moodcatHandler = new MoodcatHandler();
+        handlers.addContext("/ ", "/").setHandler(moodcatHandler);
+
+        return handlers;
+    }
+
+    /**
+     * The MoodcatServletModule is the Dependency Injection module for the
+     * MoodCat base API service. It tells Google Guice which classes (and their
+     * dependencies) to instantiate.
+     */
+    public static class MoodcatServletModule extends ServletModule {
+
+        /**
+         * The rootFolder that contains all resources.
+         */
+
+        public MoodcatServletModule() {
+        }
+
+        @Override
+        protected void configureServlets() {
+            // Bind the database module
+            this.bindDatabaseModule();
+
+            // Bind the bulk insertion
+            this.bind(BulkInsertData.class).asEagerSingleton();
+        }
+
+        private void bindDatabaseModule() {
+            install(new DbModule());
+
+            filter("/*").through(PersistFilter.class);
+            requireBinding(EntityManager.class);
+            requireBinding(EntityManagerFactory.class);
+        }
+    }
+
+    /**
+     * The handler used for injecting the database module.
+     */
+    public class MoodcatHandler extends ServletContextHandler {
```

Comment:
Maybe use an anonymous instance because the `MoodcatHandler` adds no behaviour to `ServletContextHandler` and the name `ServletContextHandler` gives more insight in what it actually is.